sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -111,9 +116,25 @@ export default function VisualSearchBar({
       return null;
     }
     const options = autocompleteResults.map(result => {
+      const verticalKeys = hideVerticalLinks ? undefined : ['people', 'events'];//result.verticalKeys;
```

Comment:
this is from testing right? If we want to keep it we can have a more descriptive comment